Makefile

```diff
@@ -100,6 +100,10 @@ build: manifests generate fmt vet ## Build manager binary.
 .PHONY: run
 run: docker-build kind-cluster kind-load cert-mgr rukpak install deploy wait ## Build the operator-controller then deploy it into a new kind cluster.
 
+.PHONY: run-local
+run-local: manifests generate fmt vet  ## Run a controller from your host. Make sure that necessary CRDs are installed in the cluster.
```

Comment:
Would it make sense to do an `install` here, so that CRDs are install on the default cluster?

Reply:
I was thinking about it, then realized that we need rukpak and cert-manager CRDs too. Which I why I excluded everything instead of selectively installing only operator CRDs.